css/style.css

```diff
@@ -1,4 +1,251 @@
+  text-decoration: none;
+  font-weight: 600;
+  font-size: 15px;
+  color: black;
+  white-space: nowrap;
+}
+
+.menubar__link:hover, 
+.menubar__link:active, 
+.menubar__link:focus {
+  cursor: pointer;
+  color: #e51937;
+  box-shadow: 0 2px 0 #e51937;
+  opacity: 0.75;
+  transition: 0.4s;
+}
+
+.menubar__link_red:nth-child(1) {
+  margin-right: 63px;
+}
+
+.menubar__link:nth-child(2) {
+  margin-right: 25px;
+}
+
+.menubar__link:nth-child(3) {
+  margin-right: 44px;
+}
+
+.menubar__link:nth-child(4) {
+  margin-right: 48px;
+}
+
+.menubar__link:nth-child(5) {
+  margin-right: 221px;
+}
+
+.menubar__link:nth-child(6) {
+  margin-right: 32px;
+  font-size: 16px;
+}
+
+.menubar__link:nth-child(7) {
+  margin-right: 32px;
+  font-size: 16px;
+}
+
+.button {
+  display: block;
+  background-color: #fff;
+  width: 24px;
+  height: 15px;
+  position: relative;
+  transition: all 0.5s ease 0s;
+}
+
+.button:nth-child(8) {
+  margin-right: 10px;
+}
+
+.button__line {
+  position: absolute;
+  width: inherit;
+  height: 2px;
+  background-color: #e51937;
+}
+
+.second {
+  top: 7px;
+}
+
+.third {
+  top: 14px;
+}
+
+.button:hover, 
+.button:focus, 
+.button:active {
+  transform: rotate(90deg);
+  cursor: pointer;
+  transition: all 0.5s ease 0s;
+}
+
+.content {
+  position: relative;
+  background: url("../img/jpg/Tesla\ logo\ car.jpg");
+  background-repeat: no-repeat;
+  background-position-y: -13px;
+  background-position-x: -46px;
+}
+
+.content__main-title, 
+.content__title {
+  text-align: center;
+  margin: 0;
+}
+
+.content__main-title {
+  font-weight: 600;
+  font-size: 90px;
+  margin-bottom: -3px;
+}
+
+.content__title {
+  font-weight: 600;
+  font-size: 16px;
+  padding-bottom: 26px;
+}
+
+.content__data-car {
+  display: flex;
+  margin: 478px 0 25px 270px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.content__facts {
+  display: flex;
+  margin-right: 115px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.data__speed-icon {
+  width: 54px;
+  height: 42px;
+  padding-right: 10px;
+}
+
+.content__road-icon {
+  width: 40px;
+  height: 59px;
+  padding-right: 10px;
+}
+
+.data__speed-block {
+  padding-right: 68px;
+}
+
+.data__figures {
+  font-weight: 600;
+  font-size: 35px;
+}
+
+.data__speed {
+  font-weight: 600;
+  font-size: 22px;
+}
+
+.data__specifications {
+  display: inline-block;
+  font-size: 13px;
+}
+
+.testdrive {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  white-space: nowrap;
+  width: 118px;
+  margin-right: 40px;
+}
+.testdrive__img {
+  width: 22px;
+  height: 22px;
+  padding-right: 5px;
+}
+
+.testdrive__text {
+  font-weight: bold;
+  font-size: 14px;
+  text-transform: uppercase;
+
+}
+
+.order-btn {
+  margin-right: 275px;
+  border: 2px solid #e51937;
+  width: 213px;
+  height: 40px;
+  background-color: #fff;
+  font-weight: bold;
+  font-size: 14px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #e51937;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)